lms/templates/problem.html

```diff
@@ -28,12 +28,13 @@ <h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describ
 
     <div class="submit-attempt-container">
       <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
-          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer. Some problems have options such as save, reset, hints, or show answer. These options follow the Submit button.")}</span>
+          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer")}</span>
       </button>
       <div class="submission-feedback" id="submission_feedback_${short_id}">
-      % if attempts_allowed:
+        % if attempts_allowed:
           ${ungettext("You have used {num_used} of {num_total} attempt", "You have used {num_used} of {num_total} attempts", attempts_allowed).format(num_used=attempts_used, num_total=attempts_allowed)}
-      % endif
+        % endif
+        <span class="sr">${_("Some problems have options such as save, reset, hints, or show answer. These options follow the Submit button.")}</span>
```

Comment:
Small nit with off screen text: when appending it or prepending it to other text, you have to make sure spaces are explicitly inserted otherwise the last word in one span will be combined without a space with the first word of the next span. Its hard to detect without a screen reader because it involves invisible text. Recommend either adding a space to the end of the line that reports remaining attempts or the beginning of the new informative text.